tasks/vmaas_sync/package_sync.go

```diff
@@ -218,15 +235,62 @@ func storePackageDetailsFrmPkgListItems(tx *gorm.DB, nameIDs map[string]int, pkg
 	}
 	utils.Log("packages", len(toStore)).Info("Collected packages to store")
 
+	err := toStore.storeOrUpdate(tx)
+	return err
+}
+
+//nolint:funlen
+func (pkgs pkgSlice) storeOrUpdate(tx *gorm.DB) error {
+	var toUpdate pkgSlice
+
+	nameIDEvra := make([][]interface{}, 0, len(pkgs))
+	toStore := make(pkgSlice, 0, len(pkgs))
+	for _, pkg := range pkgs {
+		nameIDEvra = append(nameIDEvra, []interface{}{pkg.NameID, pkg.EVRA})
+	}
```

Comment:
Maybe this would be better?
```suggestion
	for _, p := range ps {
		nameIDEVRAs = append(nameIDEVRAs, nameIDandEvra{p.NameID, p.EVRA})
	}
```

Reply:
don't work with gorm, leaving it as [][]interface